src/quo/components/wallet/wallet_activity/style.cljs

```diff
@@ -54,10 +54,17 @@
   {:margin-right 4
    :color        (colors/theme-colors colors/neutral-100 colors/white theme)})
 
-(def icon-container
-  {:width      32
-   :height     32
-   :margin-top 8})
+(defn icon-container
+  [theme]
+  {:width           33
+   :height          33
+   :padding         1
+   :border-width    1
+   :border-radius   16
+   :border-color    (colors/theme-colors colors/neutral-20 colors/neutral-80 theme)
+   :margin-top      8
+   :justify-content :center
+   :align-items     :center})
```

Comment:
The missing circle, the height was 32 but changed to 33 because the border radius